include/cuco/bucket_storage.cuh

```diff
@@ -29,200 +29,210 @@
 #include <memory>
 
 namespace cuco {
+/// Bucket type alias
+template <typename T, int32_t BucketSize>
+using bucket = detail::bucket<T, BucketSize>;
 
-/// Window type alias
-template <typename T, int32_t WindowSize>
-using window = detail::window<T, WindowSize>;
-
-/// forward declaration
-template <typename T, int32_t WindowSize, typename Extent>
-class aow_storage_ref;
+/// Alias for bucket
+template <typename T, int32_t BucketSize>
+using window = bucket<T, BucketSize>;
```

Comment:
Do we want to keep this alias for now so we don't break in a gazillion places in cudf?

Reply:
exactly